webservices/resources/reports.py

```diff
@@ -46,25 +50,35 @@ def parse_types(types):
             'type': 'string',
         },
         {
+
             'name': 'committee_type',
             'in': 'path',
             'type': 'string',
-            'description': 'House, Senate or presidential',
-            'enum': ['presidential', 'pac-party', 'house-senate'],
+            'description': 'House, Senate, presidential, independent expenditure only',
+            'enum': ['presidential', 'pac-party', 'house-senate', 'ie-only'],
         },
     ],
 )
 class ReportsView(Resource):
 
+    @schemas.marshal_with(schemas.CommitteeReportsPageSchema(), wrap=False)
```

Comment:
What would you think about putting independent expenditures under a separate endpoint: `/committee/{committee_id}/expenditures` and `/expenditures`? This class gets more complicated if we want it to handle data from both F3 and F5 filings--see `if self.committee_type == 'ie-only'` checks and these extra methods. We also lose the ability to describe the output of the resource in Swagger, since Swagger doesn't have the ability to specify that the output will belong to one of several different schemas.